src/build.py

```diff
@@ -466,6 +466,10 @@ def SyncPrebuiltCMake(name, src_dir, git_repo):
       print 'Error downloading %s: %s' % (url, e)
       raise
 
+  # Add prebuilt cmake to PATH so any subprocesses use a consistent cmake.
+  os.environ['PATH'] = (os.path.join(PREBUILT_CMAKE_DIR, 'bin') +
+                        ':' + os.environ['PATH'])
```

Comment:
let's make the ':' `os.pathsep` so there's at least a chance that it will work on windows.